src/eventsService.ts

```diff
@@ -0,0 +1,28 @@
+const GET_UPCOMING_EVENTS_URL =
+	'https://2sif0durcj.execute-api.eu-west-1.amazonaws.com/dev/get-upcoming-events';
+
+let cachedUpcomingEvents: any[];
+
+async function fetchUpcomingEvents() {
```

Comment:
Would it make sense to also fetch the previous events in this service?

Reply:
I thought about it, and I might do that even though it is only called in one spot. At least it will be more consistent.

Reply:
👍 and you can cache the result as well!